src/bin/gui/main_gui.ml

```diff
@@ -1195,33 +1193,27 @@ let start_gui all_used_context =
          ignore(toolbar#insert tool_item);
 
          let clean_button =
-           GButton.tool_button ~label:" Clean unused" ~stock:`CLEAR ()
-         in
+           GButton.tool_button ~label:" Clean unused" ~stock:`CLEAR () in
          toolbar#insert clean_button;
          clean_button#misc#hide ();
 
          let toolsearch =
-           GButton.toolbar
-             (*~tooltips:true*)
-             ~packing:(toolbox#pack ~fill:true)
-             ()
+           GButton.toolbar (*~tooltips:true*)
+             ~packing:(toolbox#pack ~fill:true) ()
          in
          toolsearch#set_icon_size `DIALOG;
 
          let tool_item = GButton.tool_item ~packing:toolsearch#insert () in
          let search_box =
-           GPack.hbox ~spacing:5 ~border_width:5 ~packing:tool_item#add ()
-         in
+           GPack.hbox ~spacing:5 ~border_width:5 ~packing:tool_item#add () in
          ignore(GMisc.image ~icon_size:`LARGE_TOOLBAR
```

Comment:
I'm not a fan of it either. I only indented that way because the rest of the file is indented like that. I'll revert the changes, but fixing the indentation and style in other places in the files should be done in a separate PR so I won't do it here.